JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -580,19 +580,28 @@ module Microsoft.ApplicationInsights {
         /**
         * Checks if a request url is not on a excluded domain list and if it is safe to add correlation headers
         */
-        public static canIncludeCorrelationHeader(config: IConfig, requestUrl: string) {
+        public static canIncludeCorrelationHeader(config: IConfig, requestUrl: string, currentHost: string) {
             if (config && config.disableCorrelationHeaders) {
                 return false;
             }
 
+            if (!requestUrl) {
+                return false;
+            }
+
+            let requestHost = UrlHelper.parseUrl(requestUrl).host.toLowerCase();
```

Comment:
you need to include `port` into comparison. As I recall

Reply:
Ideally paste a link to official documentation on how browsers compare hosts for CORS. There are official rules

Reply:
Port included in the host - I've double checked for different browsers. This comparison existed since the initial implementation (legacy headers) and here we improved it to be case insensitive.